### ISSUE TITLE
Make the game playable without mouse

### ISSUE THREAD

**Issue by holke** [youvsvirus/youvsvirus-unity#144]
The game is a keyboard game.

It should be playable with keyboard only.

This mostly means making  the menu navigatable with the keyboard.
